backend/dal/PIMSContext.cs

```diff
@@ -82,6 +82,20 @@ public class PimsContext : DbContext
         public DbSet<Workflow> Workflows { get; set; }
         public DbSet<WorkflowType> WorkflowTypes { get; set; }
         #endregion
+
+        #region Leases
+        public DbSet<Lease> Leases { get; set; }
```

Comment:
Feel like I've commented on this in another PR.  Is this PR contain commits from another PR?

Reply:
Yes, they share a commit with backend functionality.